mongodbatlas/private_endpoints.go

```diff
@@ -128,18 +142,21 @@ func (s *PrivateEndpointsServiceOp) List(ctx context.Context, groupID string, li
 	return *root, resp, nil
 }
 
-// Delete removes one private endpoint connection in an Atlas project.
-// See more: https://docs.atlas.mongodb.com/reference/api/private-endpoint-delete-one-private-endpoint-connection/
-func (s *PrivateEndpointsServiceOp) Delete(ctx context.Context, groupID, privateLinkID string) (*Response, error) {
+// Delete one private endpoint service for AWS or Azure in an Atlas project.
+// See morehttps://docs.atlas.mongodb.com/reference/api/private-endpoints-service-delete-one/
```

Comment:
[nit] same as before but this one on the typo `morehttps`
```suggestion
// Delete one private endpoint service for AWS or Azure in an Atlas project.
//
// See more: https://docs.atlas.mongodb.com/reference/api/private-endpoints-service-delete-one/
```